include/zephyr/net/wifi.h

```diff
@@ -78,6 +80,10 @@ enum wifi_security_type {
 	WIFI_SECURITY_TYPE_EAP_PEAP_TLS,
 	/** EAP TLS SHA256 security - Enterprise. */
 	WIFI_SECURITY_TYPE_EAP_TLS_SHA256,
+	/** EAP TLS SUITEB-SHA256 security - Enterprise. */
+	WIFI_SECURITY_TYPE_EAP_TLS_SUITEB_SHA256,
+	/** EAP TLS SUITEB-192 security - Enterprise. */
+	WIFI_SECURITY_TYPE_EAP_TLS_SUITEB_SHA384,
```

Comment:
why need to add specific `EAP TLS SUITEB` and `EAP TLS SUITEB-192`?
In `wifi connect` cmd, use `[-S, --suiteb-type]` to distinguish `1:suiteb, 2:suiteb-192`.
Actually, each EAP mode (`PEAPv0-MSCHAPv2, PEAPv1-GTC,  EAP-TTLS-MSCHAPv2, EAP-PEAP-TLS`) can add suiteb, not only `EAP_TLS`,

Reply:
Yes, Suite-B is applicable for all 802.1x methods. Looking at the enumeration list, we are using that for both scan and connect, and by the same logic, FT is also applicable for all security methods, no? so, should we also remove the FT and make it a separate param?

Reply:
we can keep, but also need to change `wpas_key_mgmt_to_zephyr`, right?

Reply:
Agreed, Removed the the mentioned configs and created a separate SUITE-B identifier.